src/components/group/ohri-obs-group.scss

```diff
@@ -1,48 +0,0 @@
-.flexColumn {
-  display: flex;
-  flex-direction: column;
-  margin-right: 2.5rem;
-  flex: 1;
-}
-
-.flexRow {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  width: 100%;
-}
-
-.obsGroupColumn {
-  max-width: 18rem;
-}
-
-.obsGroupColumn > div, .obsGroupColumn div[class*="cds--number"] {
-  margin-top: 0 !important;
-}
-
-/** Targets carbon's multiselect */
-.obsGroupColumn > div > div > div {
-  max-width: 18rem !important;
-}
-
-/** Targets carbon's dropdown */
-.obsGroupColumn > div > div > div > div {
-  max-width: 18rem !important;
-}
-
-.parent{
-  width: 360px;
-}
-
-.tooltipWithUnspecified{
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-}
-
-.tooltip{
-  display: flex;
-  justify-content: flex-end;
-  align-items: center;
-  margin-top: 0.5rem;
```

Comment:
So the idea of this was to undo all styling from the controls and to start the styling process from scratch centrally, as there were a lot of deprecated/irrelevant styling within the many stylesheets. I agree that the cleanup could have been handled better.

Reply:
While I'm on board with the clean-up initiative, I'd like to emphasize that it's generally considered good practice to maintain specific CSS rules within component-specific stylesheets rather than consolidating everything into one large CSS file. This approach helps to keep things more granular and makes maintenance easier in the long run.